VMBackup/main/workloadPatch/WorkloadUtils/logRestore.py

```diff
@@ -0,0 +1,114 @@
+import sys
+import threading
+import os
+from time import sleep
+import subprocess
+from datetime import datetime
+import re
+try:
+    import ConfigParser as ConfigParsers
+except ImportError:
+    import configparser as ConfigParsers
+
+class Incremental:
```

Comment:
@shridpant -rename to LogRestore also the classname to LogRestore

Reply:
Done